app/models/invoice.rb

```diff
@@ -21,4 +21,8 @@ def merchant_invoice_items(merchant)
   def potential_revenue(merchant)
     merchant_invoice_items(merchant).sum(:unit_price)
   end
+
+  def admin_potential_revenue
+    invoice_items.sum('quantity * unit_price')
```

Comment:
we might need to refactor this to look for only invoice_items that are connected to successful transactions.